src/main/java/org/sonarlint/intellij/ui/CurrentFileConnectedModePanel.java

```diff
@@ -166,18 +166,18 @@ private void switchCards() {
       runOnPooledThread(project, () -> {
         var projectBindingManager = getService(project, ProjectBindingManager.class);
         projectBindingManager.tryGetServerConnection().ifPresentOrElse(serverConnection -> {
-          try {
-            var module = SonarLintAppUtils.findModuleForFile(selectedFile, project);
-            if (module == null) {
-              switchCard(EMPTY);
-            } else {
-              connectedCard.updateTooltip(module, serverConnection);
-              switchCard(CONNECTED);
+            try {
```

Comment:
I think you could revert this change, I don't see anything new compared to the old code. Or was it badly formatted?